src/assets/CollectibleDetectionController.test.ts

```diff
@@ -429,19 +476,26 @@ describe('CollectibleDetectionController', () => {
       symbol: 'II',
       totalSupply: 10,
     };
+
+    const selectedAddress = '0x1';
     collectibleDetection.configure({
-      selectedAddress: '0x1',
+      selectedAddress,
       networkType: MAINNET,
     });
+    const { chainId } = collectibleDetection.config;
     await collectibleDetection.detectCollectibles();
     // First fetch to API, only gets information from contract ending in HH
-    expect(collectiblesController.state.collectibles).toStrictEqual([
-      collectibleHH2574,
-    ]);
+    expect(
+      collectiblesController.state.allCollectibles[selectedAddress]?.[
+        chainId
+      ] || [],
```

Comment:
what am I missing about these instances of`|| []`? I don't see why they are needed?